src/main/java/frc/robot/utils/SparkMaxUtils.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.utils;
+
+import java.util.function.BooleanSupplier;
+
+import com.revrobotics.AbsoluteEncoder;
+import com.revrobotics.REVLibError;
+
+public class SparkMaxUtils {
+    public static final int check(REVLibError error) {
+        return error == REVLibError.kOk ? 0 : 1;
+    }
```

Comment:
We don't need this right now, we aren't concerned about REVLibErrors at the moment. It might be something we add later if there's time but I don't want it to be fragmented where we have it in some places and not in others.